src/main/java/org/sefglobal/scholarx/model/Mail.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Map;
 
 @Data
```

Comment:
Remove this `@Data` annotation because we have manually defined the getters and setters.